apps/monitor/src/index.ts

```diff
@@ -1,13 +1,18 @@
-import { defineString } from 'firebase-functions/params'
+import { defineInt, defineString } from 'firebase-functions/params'
 import * as functions from 'firebase-functions/v2/scheduler'
 import { testDomainSignQuery, testPNPSignQuery } from './test'
 
 const contextName = defineString('MONITOR_CONTEXT_NAME')
 const blockchainProvider = defineString('BLOCKCHAIN_PROVIDER')
+const chainID = defineInt('CHAIN_ID')
 
 export const odisMonitorScheduleFunctionPNPGen2 = functions.onSchedule(
   'every 5 minutes',
-  async () => testPNPSignQuery(blockchainProvider.value(), contextName.value() as any),
+  async () =>
+    testPNPSignQuery(
+      { rpcURL: blockchainProvider.value(), chainID: chainID.value() as 44787 },
```

Comment:
Why are we casting this to a specific chain ID? What if we want to run the test on mainnet?